src/util/validator.js

```diff
@@ -35,27 +36,42 @@ module.exports = class Validator {
     assertCondition(t.executionCondition, 'executionCondition')
     assertString(t.expiresAt, 'expiresAt')
 
+    if (t.account) {
+      debug('Deprecation warning: switch from the "account" field to the "to" and "from" fields!')
```

Comment:
This should use Node's built-in deprecation function https://nodejs.org/api/util.html#util_util_deprecate_function_string